projector/reducer_config.go

```diff
@@ -0,0 +1,7 @@
+package projector
+
+//reducerConfig Contains all of the information needed to use a given reducer.
+type reducerConfig struct {
+	Reducer  MessageReducer
+	PollTime int64
```

Comment:
This should be of type "time.Duration" ... which is an int64, but makes it clear on how to initialize it (ie. time.Second * 5 or time.Millisecond * 100).

Reply:
+1